app/src/main/java/com/yello/presentation/onboarding/fragment/StudentID/StudentIdFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.yello.presentation.onboarding.fragment.StudentID
+
+import android.os.Bundle
+import android.view.View
+import com.example.ui.base.BindingFragment
+import com.example.ui.view.setOnSingleClickListener
+import com.yello.R
+import com.yello.databinding.FragmentStudentidBinding
+import com.yello.presentation.onboarding.fragment.StudentID.list.SearchDialogDepartmentFragment
+import com.yello.presentation.onboarding.fragment.StudentID.list.SearchDialogIDFragment
+
+class StudentIdFragment : BindingFragment<FragmentStudentidBinding>(R.layout.fragment_studentid) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initSearchIDBtnClickListener()
+        initSearchDeaprtmentBtnClickListener()
+    }
+
+    private fun initSearchIDBtnClickListener() {
```

Comment:
```suggestion
    private fun initSearchIdBtnClickListener() {
```
Fragment 형식이랑 통일하는 건 어떤가요?